<launch>
  <param name="choice_ros"  type="int" value="1"/>
  <param name="policy_path" value="$(find tobo_planner)/scripts/_policy.xml"/>
  <param name="transitions_path" value="$(find tobo_planner)/scripts/_transitions.xml"/>

  <node pkg="tobo_planner" type="planning.py" name="planning" output="screen">
  </node>

</launch>
